backend/src/controllers/advancedAnalysisController.js

```
@@ -0,0 +1,877 @@
+        const rows = [
+            ['Metric', 'Value', 'Category'],
+            ['Total Holders', data.holder_analysis?.holder_distribution?.summary?.total_holders || 0, 'Distribution'],
+            ['Total Whales', data.whale_analysis?.whale_count || 0, 'Whales'],
+            ['Market Sentiment', data.trading_insights?.overall_sentiment || 'NEUTRAL', 'Sentiment'],
+            ['Risk Level', data.trading_insights?.risk_assessment?.overall_risk || 'MEDIUM', 'Risk'],
+            ['Concentration Risk', data.trading_insights?.key_metrics?.concentration_risk || 0, 'Risk']
+        ];
+        
+        return rows.map(row => row.join(',')).join('\n');
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)